dominion/dominion_ui/src/components/Modal/GetLocation.js

```diff
@@ -13,77 +13,74 @@ const styles = {
     }
   }
 };
+const countries = [
+  {
+    name: 'Kenya',
+    url: '/profiles/country-KE'
+  },
+  {
+    name: 'South Africa',
+    url: '/profiles/country-ZA'
+  },
+  {
+    name: 'Tanzania',
+    url: '/profiles/country-TZ'
+  },
+  {
+    name: 'Nigeria',
+    url: '/profiles/country-NG'
+  }
+];
```

Comment:
The urls will have to change to `/country-nam-slug` e.g. `/south-africa` so that they go to the country page.